CheatSheets/GithubAPI.md

```diff
@@ -2,16 +2,20 @@ GitHub API
 Authentication
 
 1.Do I need to authenticate?
+
```

Comment:
Would be great to have some formatting so you can easily see topics, headers, etc.